topology-mainnet-infra.yaml

```diff
@@ -0,0 +1,8 @@
+nodes:
```

Comment:
will this deploy something or is it just for a stub?

Reply:
@domenkozar, no, just a stub, right.

Reply:
let's add a comment here to explain why this file exists for future refactoring.